src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -141,23 +141,33 @@ public static void changeDataTypesInOutputSchema(String jsonFieldList) {
    */
   public static void clickGetSchemaButton() {
     ElementHelper.clickOnElement(CdfPluginPropertiesLocators.getSchemaButton);
+    WaitHelper.waitForElementToBeDisabled(CdfPluginPropertiesLocators.getSchemaButton);
+    /*
+    TODO : Remove the commented lines of code once the bug is resolved by Chrome Team.
+    Bug : https://cdap.atlassian.net/browse/PLUGIN-1693
     WaitHelper.waitForElementToBeOptionallyDisplayed(
       CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.SMALL_TIMEOUT_SECONDS);
     WaitHelper.waitForElementToBeHidden(
       CdfPluginPropertiesLocators.locatorOfLoadingSpinnerOnGetSchemaButton(), ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
     WaitHelper.waitForElementToBeDisplayed(CdfPluginPropertiesLocators.getSchemaButton);
+    */
   }
 
   /**
    * Click on the Validate button inside Plugin's properties page
    */
   public static void clickValidateButton() {
     ElementHelper.clickOnElement(CdfPluginPropertiesLocators.validateButton);
+    WaitHelper.waitForElementToBeDisabled(CdfPluginPropertiesLocators.validateButton);
+    /*
+    TODO : Remove the commented lines of code once the bug is resolved by Chrome Team.
```

Comment:
Is there a chome bug link ? How are we sure this is a chrome issue ?

Reply:
https://bugs.chromium.org/p/chromedriver/issues/detail?id=4440 
https://cdap.atlassian.net/browse/PLUGIN-1693

Can you check both the issues , one is raised for chrome browser and recently we have raise the second one for CDAP where the error is same "No node with given id found" .